vignettes/tcplfit2-vignette.Rmd

```diff
@@ -350,4 +357,15 @@ DT::datatable(res_wide,options = list(scrollX = T))
 
 *The same hitcalling can be done with the full data set, `dat`, as well.*
 
+This output table is the same format as the `res` table in example 3. Users can use the plot code in the chunk that demonstrates the use of `concRespPlot` in example 3 to visualize fits from this output table. 
```

Comment:
Minor updates to the wording for clarity to the audience:
"The output table resulting from the code in the last code chunk is the same format as the `res` table in example 3.  Thus, users can use the `concRespPlot` function, as done previously in example 3, to plot the results.  The next code chunk demonstrates how to visualize the example 4 fit results."